lib/services/github.go

```diff
@@ -26,7 +26,11 @@ import (
 	"github.com/gravitational/teleport/lib/utils"
 )
 
-var ErrRequiresEnterprise = trace.AccessDenied("this feature requires Teleport Enterprise")
+// generic error for attempting usage of a enterprise feature
+var ErrRequiresEnterprise = trace.AccessDenied("This feature requires Teleport Enterprise")
+
+// error message for usage of endpoint urls in GitHub auth connector
+var ErrGitHubEndpointRequiresEnterprise = trace.AccessDenied("Endpoint URLs are not allowed in GitHub Auth Connector. This feature requires Teleport Enterprise")
```

Comment:
I would try to use `trace.Wrap` or `fmt.Errorf` to wrap `ErrRequiresEnterprise` so that `errors.Is` checks still succeed.

Maybe something like:

```
return nil, fmt.Errorf("GitHub enterprise endpoints: %w", ErrRequiresEnterprise)

```